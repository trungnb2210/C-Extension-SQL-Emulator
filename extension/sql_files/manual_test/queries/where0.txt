SELECT object, price
FROM prices
WHERE price > 100
