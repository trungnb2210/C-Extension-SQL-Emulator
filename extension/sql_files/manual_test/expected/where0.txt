QUERY RESULT:
  object   price 
------------------
| Box    | 250   | 
| Tire   | 120   | 
| Table  | 300   | 
------------------
